carla_integration/VehicleConfigParams.yaml

```diff
@@ -0,0 +1,52 @@
+# Defines the ros parameters which define the characteristics of this host vehicle configuration
+
```

Comment:
You need to provide the make model and year as well
```
# String: Host vehicle make
vehicle_make: 'Lexus'

# String: Host vehicle model
vehicle_model: 'RX-450'

# Integer: Host vehicle year
# Units: Year
vehicle_year: 2019
```

Reply:
Are these the requirements? Where are they used?

Reply:
I think they are used in the BSM generator. But everything in this file is expected to be available to the system, so they need to be included. It shouldn't cause an issue if it disagrees with the simulated vehicle type, but if you know the type you could also change them

Reply:
Added vehicle_model and vehicle_make

Reply:
You should also add vehicle_year

Reply:
Added